src/main/java/com/example/sharemind/chatMessage/presentation/ChatMessageController.java

```diff
@@ -0,0 +1,47 @@
+package com.example.sharemind.chatMessage.presentation;
+
+import com.example.sharemind.chat.application.ChatService;
+import com.example.sharemind.chatMessage.application.ChatMessageService;
+import com.example.sharemind.chatMessage.dto.request.ChatMessageCreateRequest;
+import com.example.sharemind.chatMessage.dto.response.ChatMessageResponse;
+import java.util.Map;
+import java.util.Objects;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.simp.SimpMessageHeaderAccessor;
+import org.springframework.stereotype.Controller;
+
+@Slf4j
+@RequiredArgsConstructor
+@Controller
+public class ChatMessageController {
+
+    private final ChatService chatService;
+    private final ChatMessageService chatMessageService;
+
+    private void handleChatMessage(Long chatId, ChatMessageCreateRequest request,
+                                   SimpMessageHeaderAccessor headerAccessor, boolean isCustomer) {
+        Map<String, Object> sessionAttributes = Objects.requireNonNull(headerAccessor.getSessionAttributes());
+
+        chatService.validateChat(sessionAttributes, chatId);
+
+        String nickName = (String) sessionAttributes.get("userNickname");
+        chatMessageService.createAndSendChatMessage(request, chatId, isCustomer, nickName);
+    }
+
+    @MessageMapping("/chatMessage/customers/{chatId}")
```

Comment:
다른 컨트롤러들과 주소 형식 통일되면 좋을 것 같습니다!
```suggestion
    @MessageMapping("/api/v1/chatMessages/customers/{chatId}")
```